src/NServiceBus.AcceptanceTests/Routing/When_base_event_from_2_publishers.cs

```diff
@@ -13,18 +13,10 @@ public class When_base_event_from_2_publishers : NServiceBusAcceptanceTest
         {
             var context = await Scenario.Define<Context>()
                .WithEndpoint<Publisher1>(b =>
-                        b.When(c => c.SubscribedToPublisher1, bus =>
-                        {
-                            bus.Publish(new DerivedEvent1());
-                            return Task.FromResult(0);
-                        })
+                        b.When(c => c.SubscribedToPublisher1, bus => bus.PublishAsync(new DerivedEvent1()))
```

Comment:
Don't we need to `await` here? (or is the acpt test fx awaiting the lambda behind the scenes)